src/ensembl/src/shared/components/communication-framework/contact-us/contact-us-form/initial/ContactUsInitialForm.tsx

```diff
@@ -209,15 +255,46 @@ const ContactUsInitialForm = () => {
         </div>
 
         <div className={commonStyles.submit}>
-          {exceedsAttachmentsSizeLimit(state) && (
-            <span className={commonStyles.errorText}>
-              Attachment(s) exceed 10 MB
-            </span>
+          {!isFormValid && (
+            <div className={commonStyles.formErrors}>
```

Comment:
The div that shows the error is there in the DOM until the form is enabled.
When the form is enabled this is removed and you will see the bottom div moving upwards due to the margin on this div (12px) https://github.com/Ensembl/ensembl-client/pull/549/files#diff-1d4823aa7281cf982010dd4c2118669983f65c0ec6a2c85f24f5875a4aabdd75R70

Reply:
I've updated the styles. The vertical shift of the slide button now only happens when there is an actual error to display above it. It's quite likely that Andrea will update the design to deal with the errors.